packages/webdriverio/src/commands/browser/reloadSession.ts

```diff
@@ -11,6 +11,14 @@ const log = logger('webdriverio')
  * Be careful though, this command affects your test time tremendously since spawning
  * new Selenium sessions is very time consuming especially when using cloud services.
  *
+ * Connection parameters such as hostname, port, protocol, etc. can be added along side
+ * browserName when you want to connect to a different remote service. This is useful
+ * in a situation, for example, where you start a test in native app and need to verify
+ * data in web app.
```

Comment:
Just for understanding: this would completely get rid of the mobile session so you couldn't continue anything on the native app side, correct?

Reply:
Yeah, I tested it out locally. This is not meant to be for concurrent session like multi-remote, if that's what you're asking. It's meant to be used within a single session. Btw, I made a small change again, I removed deepmerge because I don't want caps from original session to be mixed with a new one.